pkg/apis/v1alpha1/common.go

```diff
@@ -40,6 +75,8 @@ func GetAPITemplate(templateKind string) (client.Object, error) {
 		template = &ClusterConfigTemplate{}
 	case "ClusterTemplate":
 		template = &ClusterTemplate{}
+	case "ClusterDeploymentTemplate":
```

Comment:
per-RFC, it should actually be `ClusterDeployTemplate` 

https://github.com/vmware-tanzu/cartographer/blob/a90f8c8571a47d72f964b94ca77da5ca7108367a/rfc/rfc-0013-delivery.md?plain=1#L23

is there a particular reason for changing or slip/typo?